src/Scene/LayersConfiguration.js

```diff
@@ -18,6 +18,12 @@ function LayersConfiguration() {
     this.geometryLayers = [];
     this.colorLayers = [];
     this.elevationLayers = [];
+    this.lightingLayers = [];
```

Comment:
The shader code doesn't support more than 1 light. So planning for more lights might be overkill.

Even if you want to make sure we support more than 1 light,  all of this can still be achieved without adding a `lightingLayer` (e.g: an array of lights in `Scene`).

Can you state why you believe that defining "light layers" is the best place for this?